source/agora/script/Engine.d

```diff
@@ -493,6 +494,36 @@ public class Engine
                     return "VERIFY_SIG signature failed validation";
                 break;
 
+            case OP.VERIFY_HEIGHT_LOCK:
+                if (stack.count() < 1)
```

Comment:
`if (stack.empty)`
Does this check the same as this?

Reply:
Oh, good eye! I will fix it up.